Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/CategoryContainer.cpp

```diff
@@ -0,0 +1,43 @@
+#include "CategoryContainer.h"
+FCategoryContainer::FCategoryContainer()
+{
+	CategoriesCount = 0;
+}
+int FCategoryContainer::GetCategoryCount()const 
+{
+	return CategoriesCount;
+}
+void FCategoryContainer::AddCategory(FCategory NewCategory)
+{
+	Categories[CategoriesCount] = NewCategory;
+	CategoriesCount += 1;
+}
+bool FCategoryContainer::IsValidIndex(int Index)
+{
+	if (!std::cin.good() || Index > 4 || Index < 0)return false;
+	else return true;
+	
+}
+void FCategoryContainer::EliminateCategory(int EliminationIndex)
```

Comment:
This method is not working as intended. What if you had 4 categories and you tried to eliminate the one at index 0? Or if you tried to eliminate index 3